src/puppetlabs/puppetdb/command.clj

```diff
@@ -97,8 +97,8 @@
 
 (def supported-command-versions
   {"replace facts" (version-range 2 4)
-   "replace catalog" (version-range 4 7)
-   "store report" (version-range 3 6)
+   "replace catalog" (version-range 4 8)
```

Comment:
Either way, we should be stating things are deprecated if we can, either fixed in this patch or another, otherwise we'll find people might miss the message.